src/main/java/com/example/notes/dto/NoteDto.java

```diff
@@ -3,10 +3,13 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 
+import java.time.LocalDateTime;
+
 @Data
 @AllArgsConstructor
 public class NoteDto {
 
     private String content;
+    private LocalDateTime createdAt;
```

Comment:
Let's add a @Builder on top of it?

Reply:
Consider to use the Timestamp instead of the LocalDateTime

Reply:
Done